internal/armada/server/lease.go

```diff
@@ -38,6 +45,13 @@ type AggregatedQueueServer struct {
 	eventStore               repository.EventStore
 	schedulingInfoRepository repository.SchedulingInfoRepository
 	decompressorPool         *pool.ObjectPool
+	// Map from "executorId-queue" to the most recent cluster usage report for that cluster.
+	// TODO: These should be stored in a database instead of in-memory.
+	// usageByExecutorId map[string]*schedulerobjects.QueueClusterResourceUsage
+	usageByQueueAndExecutor map[string]map[string]*schedulerobjects.QueueClusterResourceUsage
```

Comment:
We'll need to do this before we merge to master won't we- I presume we'll just use redis initially?